apps/server/src/modules/board/gateway/api-test/board-collaboration.gateway.spec.ts

```diff
@@ -58,6 +58,7 @@ describe(BoardCollaborationGateway.name, () => {
 		const cardNodes = cardNodeFactory.buildList(2, { parent: columnNode });
 
 		await em.persistAndFlush([columnBoardNode, columnNode, columnNode2, ...cardNodes]);
+		await em.persistAndFlush(cardNodes);
```

Comment:
this shouldnt be needed, since its already done in the line above

Reply:
true. removed.